examples/combined_ODNP.py

```diff
@@ -298,15 +319,16 @@
             SW_kHz=config_dict["SW_kHz"],
             ret_data=DNP_data,
         )
-        time_axis_coords[j + 1]["stop_times"] = time.time()
-    DNP_data.chunk("t", ["ph1", "t2"], [len(Ep_ph1_cyc), -1])
-    DNP_data.setaxis("ph1", Ep_ph1_cyc / 4)
-    DNP_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-    DNP_data.reorder(["ph1", "nScans", "t2"])
-    DNP_data.name(config_dict["type"])
+        time_axis_coords[j + thermal_scans]["stop_times"] = time.time()
     DNP_data.set_prop("stop_time", time.time())
     DNP_data.set_prop("postproc_type", "spincore_ODNP_v4")
     DNP_data.set_prop("acq_params", config_dict.asdict())
+    DNP_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
+    if phase_cycling:
+        DNP_data.chunk("t", ["ph1", "t2"], [len(Ep_ph1_cyc), -1])
+        DNP_data.setaxis("ph1", Ep_ph1_cyc / 4)
+        DNP_data.reorder(["ph1", "nScans", "t2"])
+    DNP_data.name(config_dict["type"])
```

Comment:
Why does this stuff get added here? Shouldn't it have already existed once you're all done collecting data?

Reply:
why is it getting added to this pull request?  why did it not already exist?

Reply:
does this increase the version number for the DNP data format?

Reply:
not sure what you mean by version number but the postproc does not change and is suited for this

Reply:
the version number is recorded in the filename and increases there - not the nodename if that's what you mean? e.i. "230413_27mM_ODNP_1/ODNP" for the first run then "230413_27mM_ODNP_2/ODNP" for the 2nd